src/index.html

```diff
@@ -1,13 +1,457 @@
+              <picture class="detail__photo">
+                <source
+                  media="(max-width: 743px)"
+                  srcset="./images/photos/5.png"
+                  class="detail__image"
+                >
+                <source
+                  media="(max-width: 1259px)"
+                  srcset="./images/photos/15.png"
+                  class="detail__image"
+                >
+                <source
+                  media="(min-width: 1260px)"
+                  srcset="./images/photos/25.png"
+                  class="detail__image"
+                >
+                <img
+                  src="./images/photos/5.png"
+                  alt="Auto Unlock"
+                  class="detail__image"
+                >
+              </picture>
+
+              <picture class="detail__photo detail__photo--stretched">
+                <source
+                  media="(max-width: 743px)"
+                  srcset="./images/photos/6.png"
+                  class="detail__image"
+                >
+                <source
+                  media="(max-width: 1259px)"
+                  srcset="./images/photos/16.png"
+                  class="detail__image"
+                >
+                <source
+                  media="(min-width: 1260px)"
+                  srcset="./images/photos/26.png"
+                  class="detail__image"
+                >
+                <img
+                  src="./images/photos/6.png"
+                  alt="Auto Unlock"
+                  class="detail__image"
+                >
+              </picture>
+            </div>
+
+            <div class="detail__title">
+              Hydraulic disc brakes & Lightweight
+            </div>
+
+            <div class="detail__description">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+
+          <a href="#" class="button">
+            Explore
+          </a>
+        </section>
+
+        <section class="page__section" id="contacts">
+          <h2 class="page__section-title">
+            Contact Us
+          </h2>
+          <div class="contacts">
+            <form action="#form" class="form">
+              <input
+                type="text"
+                name="name"
+                placeholder="Name"
+                class="form__field"
+                required
+              >
+
+              <input
+                type="email"
+                name="email"
+                placeholder="Email"
+                class="form__field"
+                required
+              >
+
+              <textarea
+                name="message"
+                placeholder="Message"
+                class="form__field form__field--textarea"
+                required
+              >
+              </textarea>
```

Comment:
You have a blank space between tags, that's why your placeholder is not visible
![image](https://user-images.githubusercontent.com/37161315/197525106-80dd9605-24ef-4e14-bac6-0c56f149fbc0.png)
```suggestion
              <textarea
                name="message"
                placeholder="Message"
                class="form__field form__field--textarea"
                required
             ></textarea>
```